podcasts/Referrals/ReferralCardView.swift

```diff
@@ -0,0 +1,43 @@
+import SwiftUI
+
+struct ReferralCardView: View {
+    let numberOfDaysOffered: Int
+
+    var body: some View {
+        Rectangle()
+            .cornerRadius(Constants.cardRadius)
+            .foregroundColor(Constants.cardBackgroundColor)
+            .overlay(alignment: .bottomLeading) {
+                Text(L10n.referralsGuestPassOffer(numberOfDaysOffered))
+                    .font(size: 12, style: .body, weight: .semibold)
+                    .foregroundColor(.white)
+                    .padding()
+
+            }
+            .overlay(alignment: .topTrailing) {
+                Image("plusGold")
+                    .renderingMode(.template)
+                    .resizable()
+                    .aspectRatio(contentMode: .fill)
+                    .frame(width: 12, height: 12)
```

Comment:
Should this be in Constants? I'm not really sure I like it any better having so many Constants and just jumping between lines but I noticed the `defaultCardSize` defined as a constant in another file.l

Reply:
Addressed here: https://github.com/Automattic/pocket-casts-ios/pull/2120/commits/2b0689fc534d8c9a0d21052559fb503fb7ef368d

For the record I'm also a bit split when to use the enum Constants or not, I like to use them when the value is used in multiplace spaces or to make clear the intention of the value.